Sources/WebSocket/WebSocketCloseCode+NWError.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import Network
+import WebSocketProtocol
+
+extension WebSocketCloseCode {
+    var error: NWError? {
+        switch self {
+        case .invalid:
+            return nil
+        case .normalClosure:
+            return nil
+        case .goingAway:
+            return nil
+        case .protocolError:
+            return .posix(.EPROTO)
+        case .unsupportedData:
+            return .posix(.EBADMSG)
+        case .noStatusReceived:
+            return nil
+        case .abnormalClosure:
+            return nil
+        case .invalidFramePayloadData:
+            return nil
+        case .policyViolation:
+            return nil
+        case .messageTooBig:
+            return .posix(.EMSGSIZE)
+        case .mandatoryExtensionMissing:
+            return nil
+        case .internalServerError:
+            return nil
+        case .tlsHandshakeFailure:
+            return .tls(errSSLHandshakeFail)
+        }
+    }
```

Comment:
Is it important to map all of these eventually or will they stay `nil`?

Reply:
To be honest, I'm not even sure if it's correct to map these errors. These were the only applicable `NWError`s I could find for the WebSocket-specific ones.

This is currently used to determine whether or not we should send a `.finished` or `.failure` completion to the `WebSocket`'s subscriber.